src/uct/ib/rc/accel/rc_mlx5.inl

```diff
@@ -181,6 +240,55 @@ uct_rc_mlx5_iface_common_data(uct_rc_mlx5_iface_common_t *iface,
     return hdr;
 }
 
+static UCS_F_ALWAYS_INLINE void*
+uct_rc_mlx5_iface_tm_common_data(uct_rc_mlx5_iface_common_t *iface,
+                                 struct mlx5_cqe64 *cqe,
+                                 unsigned *byte_count, unsigned *flags,
+                                 uct_rc_mlx5_mp_context_t **context_p)
+{
+    uct_ib_mlx5_srq_seg_t *seg;
+    void *hdr;
+    unsigned byte_len;
+    int stride_idx;
+    static uct_rc_mlx5_mp_context_t dummy_context;
+
+    if (!UCT_RC_MLX5_MP_ENABLED(iface)) {
```

Comment:
for non MP case we do - we call `uct_rc_mlx5_iface_common_data`, which checks inline first.
For MP we have to check message context in any case, because it can be a last (inlined) fragment of a big message

Reply:
ok